Test/coverage/verificationCoverage.bpl

```diff
@@ -1,18 +1,94 @@
 // RUN: %boogie "%s" > "%t.plain"
 // RUN: %diff "%s.expect" "%t.plain"
 // RUN: %boogie -trackVerificationCoverage -trace "%s" > "%t.coverage"
-// RUN  %OutputCheck "%s" --file-to-check="%t.coverage"
-// CHECK: Covered elements: a0, assert_a0, assert_r0, r0
-// CHECK: Covered elements: sinv_not_1$established, sinv_not_1$maintained, sinv1$assume_in_body, sinv1$established, sinv1$maintained, sinv2$assume_in_body, sinv2$established, sinv2$maintained, spost, spre1
-// CHECK: Covered elements: cont_assume_1, cont_assume_2
-// CHECK: Covered elements: false_req
-// CHECK: Covered elements: cont_req_1, cont_req_2
-// CHECK: Covered elements: assumeFalse
-// CHECK: Covered elements: tee0, tee1, ter0
-// CHECK: Covered elements: call2_tee1, tee_not_1, tee0$call1$ensures, tee0$call2$ensures, tee1$call2$ensures, ter0$call1$requires, ter0$call2$requires, ter1, xy_sum
-// CHECK: Covered elements: a_gt_10, constrained, x_gt_10
-// CHECK: Covered elements: cont_ens_abs$call_cont$ensures, xpos_abs$call_cont$requires, xpos_caller
-// CHECK: Elements covered by verification: a_gt_10, a0, assert_a0, assert_r0, assumeFalse, call2_tee1, constrained, cont_assume_1, cont_assume_2, cont_ens_abs$call_cont$ensures, cont_req_1, cont_req_2, false_req, r0, sinv_not_1$established, sinv_not_1$maintained, sinv1$assume_in_body, sinv1$established, sinv1$maintained, sinv2$assume_in_body, sinv2$established, sinv2$maintained, spost, spre1, tee_not_1, tee0, tee0$call1$ensures, tee0$call2$ensures, tee1, tee1$call2$ensures, ter0, ter0$call1$requires, ter0$call2$requires, ter1, x_gt_10, xpos_abs$call_cont$requires, xpos_caller, xy_sum
+// RUN: %OutputCheck "%s" --file-to-check="%t.coverage"
+// CHECK: Proof dependencies:
```

Comment:
Side note: if you're checking for the literal output line-by-line without anything expected in between, it's more strict and probably more efficient to use `CHECK-NEXT:` - https://github.com/stp/OutputCheck#check-next-regex

Reply:
Ah, yeah, it would probably make sense for some (but not all) of these to be `CHECK-NEXT`. However, I don't imagine the lack of that would cause us to miss bugs in this case.